app/src/main/java/com/vero/tasky/agenda/data/local/dao/DeletedAgendaItemDao.kt

```diff
@@ -0,0 +1,21 @@
+package com.vero.tasky.agenda.data.local.dao
+
+import androidx.room.*
+import com.vero.tasky.agenda.data.local.entities.DeletedAgendaItemEntity
+
+@Dao
+interface DeletedAgendaItemDao {
```

Comment:
Personally, I have a table for each item
One for reminder
One for task
One for event

If not, one table has access to every single attribute, making it a "god table"